modules/apps/commerce/commerce-payment-service/src/main/java/com/liferay/commerce/payment/internal/gateway/CommercePaymentGatewayImpl.java

```diff
@@ -98,10 +144,34 @@ public CommercePaymentEntry cancel(
 			throw new UnsupportedOperationException();
 		}
 
+		Map<String, Serializable> configurationValuesMap =
```

Comment:
Is it not possible to extract these rows (until cancelResult variable) on a private method? Because they look the same in all the methods (authorize / cancel / capture / refund)